test/integration/fixtures/runner/events-retries.fixture.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+var Runner = require('../../../../lib/runner.js');
+var assert = require('assert');
+
+var emitOrder = [
+  'suite'/* incorrect order*/, 'start', 'suite',
```

Comment:
This order changes when 00ca06b is removed?

Reply:
yes, see #3414 which shows event order before and after merging 00ca06b.